src/components/Landing/index.js

```diff
@@ -6,6 +6,7 @@ import executeElm from "../../utils/executeELM";
 import * as landingUtils from "./utility";
 import { datishFormat } from "../../helpers/formatit";
 import {
+  addMamotoTracking,
```

Comment:
@achen2401  it's Matomo, not Mamoto - can you please edit so it's easier to find :)

Reply:
thank you @mcjustin for noticing the typo! I fixed it now.